crates/sargon-uniffi/src/profile/mfa/secured_entity_control/secured_entity_control.rs

```diff
@@ -17,4 +23,8 @@ pub struct SecuredEntityControl {
     /// The believed-to-be-current security structure of FactorInstances which
     /// secures this entity.
     pub security_structure: SecurityStructureOfFactorInstances,
+
+    /// A provisional new security structure configuration which user
+    /// is about to change to
+    pub provisional: Option<ProvisionalSecurifiedConfig>,
```

Comment:
Alternatives to `provisional`:
* `draft` - discarded since it sounds like it is not "saved" or otherwise incomplete. 
* `pending` - make it sound like it has been submitted to the network and waiting for a tx complete
* `user_intention` - some version of this might be ok.

Definition of `provisional`: `"arranged or existing for the present, possibly to be changed later."`

Reply:
`Temporal`?

Reply:
You mean `temporary` I think? Hmm `temporary` relates more to time, whereas I'm going for a term which relates more to step in a sequence of steps, which I feel is a little bit more precise. And "temporary" also has a meaning that after some time it is no longer relevant, whereas what is actually happening here is that it holds the state which will be the future state, so not at all temporary, just that we are note quite there yet.